source/classes.tex

```diff
@@ -2500,7 +2500,7 @@
 };
 
 void f(Y& a) {
-  if (a) {          // ill-formed: \tcode{X::operator int()} or \tcode{Y::operator char()}
+  if (a) {          // error: \tcode{X::operator int()} or \tcode{Y::operator char()}
```

Comment:
// error: ambiguous between `X::operator int()` and `Y::operator char()`

maybe?

Reply:
Fixed.